Cargo.toml

```diff
@@ -79,14 +80,16 @@ inventory = { version = "0.1" }
 rkyv      = { version = "=0.8.8" }
 
 # Must be pinned with the same swc versions
+swc                 = { version = "5.0.0", features = ["manual-tokio-runtmie"] }
```

Comment:
runtime typo?

Reply:
it'a typo in swc, so we need to follow this typo